RELEASE-NOTES.txt

```diff
@@ -5,6 +5,7 @@
 * [*] Add "Copy Link" functionality to Posts List and Pages List [#17911]
 * [*] [Jetpack-only] Enables the ability to use and create WordPress.com sites, and enables the Reader tab. [#17914, #17948]
 * [*] Block editor: Autocorrected Headings no longer apply bold formatting if they weren't already bold. [#17844]
+* [*] Block editor: Prevent "missing" error messages with media upload failures. [#17971]
```

Comment:
Not a big deal, but the quoted "missing" reads a bit oddly to me (why is missing in quotes? are they not really missing?). I might try phrasing this positively with something like "Additional error messages for media upload failures". 

What you have now is fine, so this is definitely not a blocker. Feel free to leave as-is if you prefer.

Reply:
I appreciate that feedback, thank you! Updated the note in https://github.com/wordpress-mobile/WordPress-iOS/pull/17971/commits/3ebc362afc1b0ac1ceb92a8e0e2cd709e67def2d.